### ISSUE TITLE
Sponsorblock not working on WebOS 3 with 0.3.4

### ISSUE THREAD

**Issue by igorkulman** [webosbrew/youtube-webos#162]
While 0.3.4 fixed the syntax error on WebOS 3 so the app now load properly and ad blocking works sponsorblock is not working. 

The green button menu works and I have skipping all the sponsorblock stuff enabled but using 0.3.4 I noticed I never get a sponsorblock skip. Then I noticed that no matter what video I open there are never any sponsorblock segments on the timeline, I think they do not even load.

I compared specific videos on the Youtube website with the Sponsorblock extension and in 0.3.4 and yes, looks like 0.3.4 just never loads the sponsorblock data, they never appear on the timeline anymore.

**Comment by throwaway96** [comment]
Have you tried with older YTAF releases?

**Comment by igorkulman** [comment]
> Have you tried with older YTAF releases?

Yes, everything has worked up until 0.3.2 (including). Then the 0.3.3 did not work at all on WebOS 3 (https://github.com/webosbrew/youtube-webos/issues/151) and with the 0.3.4 build that fixes the syntax error on WebOS 3 I get working ad blocking but not working sponsorblock.

I tried the [v0.3.2-webkit1](https://github.com/throwaway96/youtube-webos/releases/tag/v0.3.2-webkit1) build from https://github.com/webosbrew/youtube-webos/issues/160 and in this one sponsorblock works just fine.

My guess is that in 0.3.4 the main syntax error that prevented the app working at all is fixed but there is still something that prevents just sponsorblock working on WebOS 3.

**Comment by hamid-elaosta** [comment]
My kids updated my rooted LG B7V and I lost root. Using the desktop manager and developer mode I installed this again but I don't get the green button, it does nothing, and no blocking of anything.

I'm not sure how much of the loss in functionality is due to loss of root, and how much is this version no working.

**Comment by throwaway96** [comment]
@hamid-elaosta

It has nothing to do with root. Use 0.3.4.

**Comment by hamid-elaosta** [comment]
> @hamid-elaosta
> 
> It has nothing to do with root. Use 0.3.4.

Thanks, I didn't realise the desktop manager wasn't deploying the latest and didn't think to check the version. Will give it a go when I'm at my PC.

**Comment by MagicLegend** [comment]
I think I'm having the same issue. Sponsorblock settings are visible with the green button, but no actual sponsorblock data is loaded. Videos that have entries on my desktop are still fully playing on the TV. Anything else I can do to diagnose this?

**Comment by marcussacana** [comment]
The sponsorblock used by the app api is down, that's should be the problem.
https://github.com/webosbrew/youtube-webos/blob/main/src/sponsorblock.js#L39
![image](https://github.com/user-attachments/assets/5ed8218e-22b7-45aa-85f7-d3e23ec26cd9)

I will try fix in my fork at least, but I don't share the binaries.

**Comment by fire332** [comment]
Does this still happen with v0.3.5-rc1?

**Comment by igorkulman** [comment]
> Does this still happen with v0.3.5-rc1?

Just tried 0.3.6 and Sponsorblock works again, thanks.

**Comment by marcussacana** [comment]
@fire332 do you know what exactly fixed this issue? I wanted do it in my fork too
